lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -145,7 +145,7 @@ $progress-bar-layers: (
 
 	&.-compact {
 		#{$self}__labelText {
-			@include FormLabelSBold();
+			@include FormLabelMBold();
```

Comment:
according to https://bethink.atlassian.net/jira/software/c/projects/IT/boards/1?modal=detail&selectedIssue=IT-4948&sprint=1798 it shouldn't be `Form` token

Reply:
True, fixed.